html/semantics/scripting-1/the-script-element/script-charset-02.html

```diff
@@ -23,7 +23,7 @@
   <script>
   test(function() {
     //these strings should not match, since the tested file is in windows-1250, and fallback is defined as utf-8
```

Comment:
This comment and the TODO at the top of this file is wrong about the fallback encoding. It's windows-1252 for this test on my locale, but can be other values. To avoid conflating tests for script encoding with the default document encoding, I suggest simply declaring the encoding at the top of the file. I suggest `<meta charset="windows-1250">` (rather than 1252) so that one of the scripts is decoded correctly, and here, `assert_equals(document.characterSet, "windows-1250")` as a precondition for the test.

With those changes, the main value of this test will that even if the script can be decoded as UTF-8, it will still be decoded using the document's encoding.